apis/helm/release/v1alpha1/types.go

```diff
@@ -40,7 +40,7 @@ type NamespacedName struct {
 
 // DataKeySelector defines required spec to access a key of a configmap or secret
 type DataKeySelector struct {
-	NamespacedName `json:",inline,omitempty"`
+	NamespacedName `json:",inline"`
```

Comment:
Why this change?

Reply:
I think this was automatically removed after I ran `make generate`. Somebody forgot to run it?